style.css

```diff
@@ -0,0 +1,146 @@
+
+nav {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+.logo {
+  margin-right: auto;
+  align-items: center;
+  color: #6070ff;
+  text-align: center;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+}
+
+.social {
+  list-style-type: none;
+  display: flex;
+}
+
+nav li {
+  list-style: none;
+  display: inline-block;
+  padding: 0 10px;
+  color: #344563;
+}
+
+.social li {
+  display: inline;
+  margin: 10px;
+}
+
+header {
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
Hey. 👋🏽 make sure you use the right spacing here to make this match the Figma while allowing your design to breathe. This section lacks margins and padding, you can see the expected padding/margin values by hovering over the contents on the Figma. See below the differences 👇🏽 

## Figma
![Screenshot 2023-02-18 at 08 55 02](https://user-images.githubusercontent.com/45739963/219869628-1a16e833-d4f1-4984-a884-d513a23ce043.png)

## Your design
![Screenshot 2023-02-18 at 08 55 27](https://user-images.githubusercontent.com/45739963/219869649-f6baf651-e30d-444f-bd8c-4ff38bdd3296.png)

A recruiter likes a good portfolio and this is a huge step toward finding your dream job. Keep going, you rock 💪🏽